staging/src/k8s.io/client-go/metadata/metadata_test.go

```diff
@@ -55,7 +56,7 @@ func TestClient(t *testing.T) {
 	testCases := []struct {
 		name    string
 		handler func(t *testing.T, w http.ResponseWriter, req *http.Request)
-		want    func(t *testing.T, client *Client)
+		want    func(t *testing.T, ctx context.Context, client *Client)
```

Comment:
Typically `ctx` is the first parameter in a function. I prefer to follow that also here and below.

Reply:
I agree, it's a bit subjective. I got annoyed by the separate parameters and proposed a solution here (still to be implemented, comments welcome): https://github.com/kubernetes/kubernetes/pull/122148#discussion_r1411874086